prototypes/as2/.gitignore

```diff
@@ -0,0 +1,21 @@
+# Don't checkin secrets
```

Comment:
Before building a real code, I created a prototype to learn how to work with AS2. Here's the prototype.

Reply:
Nice.   Can you present at a Monday Engineering meeting?   Also, any general rules gleaned about when a developer should choose to create a prototype, and any process patterns / best practices for doing a prototype?